scipy/spatial/transform/rotation.py

```diff
@@ -1620,6 +1620,13 @@ def match_vectors(cls, a, b, weights=None, normalized=False):
 
         B = np.einsum('ji,jk->ik', weights[:, None] * a, b)
         u, s, vh = np.linalg.svd(B)
+
+        # Correct improper rotation if necessary (as in Kabsch algorithm)
+        d = np.linalg.det(u @ vh) < 0
```

Comment:
Minor point, but any reason not to forego the variable and just do:
`if np.linalg.det(u @ vh) < 0: `

Reply:
Good idea.  I have made the change.